client/src/child/ListItemComponent.tsx

```diff
@@ -98,78 +100,51 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
-function getSteps() {
-  return ["Neu", "Aktiv", "Prüfen", "Erledigt"];
-}
-
-export function ChildChoreItem({
-  chore,
-  onUpdateState,
-}: ChildChoreItemprops): JSX.Element {
+export function ListItemComponent(props: Props) {
   const classes = useStyles();
-  const steps = getSteps();
-  let buttonText = "Start";
 
-  function assignmentStateToStepper(chore: ChoreWithAssignments): number {
-    let steperState = 0;
-    switch (chore.assignments[0].state) {
-      case AssignmentState.Assigned: {
-        steperState = 0;
-        buttonText = "Start";
-        break;
-      }
-      case AssignmentState.Active: {
-        steperState = 1;
-        buttonText = "Prüfen";
-        break;
-      }
-      case AssignmentState.RequestedToCheck: {
-        steperState = 2;
-        buttonText = "Warten";
-        break;
-      }
-      case AssignmentState.CheckRefused: {
-        steperState = 1;
-        buttonText = "Prüfen";
-        break;
-      }
-      case AssignmentState.CheckConfirmed: {
-        steperState = 3;
-        buttonText = "Archiv";
-        break;
-      }
-      case AssignmentState.Archived: {
-        steperState = 4;
-        buttonText = "Fertig";
-        break;
-      }
-    }
-    return steperState;
+  const subHeader = new Date(props.item.dueDate).toLocaleDateString("de-DE", {
```

Comment:
Vielleicht können wir bei Gelegenheit die Formatierung der Datums (Datümmer, Datis, wie auch immer 🤔 😉) noch irgendwo in eine Hilfsfunktion auslagern und dann sowohl im Kinder- als auch im Eltern-Dashboard wiederverwenden.

Reply:
Stimmt, würde mittlerweile wohl Sinn machen 👍